indico/modules/rb_new/client/styles/palette.scss

```diff
@@ -22,10 +22,19 @@ $highlight-color: #7dcfb6;
 $dark-highlight-color: #45ba97;
 $primary-color: #2185d0;
 
-$pre-booking-stripe-colors: ($orange, lighten($orange, 14%));
-$pre-booking-conflict-stripe-colors: ($red, lighten($red, 14%));
-$blocking-stripe-colors: (lighten($light-red, 10%), $light-red);
-$rejection-stripe-colors: (lighten($red, 0%), lighten($red, 10%));
+// Timeline colors
+$available: #07e04b;
+$booking: #fb3;
+$conflict: #f44;
+$cancellation:  #f9e3cf;
```

Comment:
one space too much here!

BTW, what about using `-color` for the variable names?

Reply:
good idea :)